products/bazel.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Bazel
+category: app
+tags: google build-tool java-runtime
+iconSlug: bazel
+permalink: /bazel
+versionCommand: bazel --version
+releasePolicyLink: https://bazel.build/release
+changelogTemplate: https://github.com/bazelbuild/bazel/blob/__LATEST__/CHANGELOG.md
+changelogTemplate: "https://github.com/bazelbuild/bazel/releases/tag/__LATEST__"
+releaseDateColumn: true
+activeSupportColumn: true
+
+auto:
+-   git: https://github.com/bazelbuild/bazel.git
+
+releases:
+-   releaseCycle: "7"
+    releaseDate: 2023-12-11
```

Comment:
According to https://bazel.build/release all cycles are LTS releases, set the flag for all cycles...
```suggestion
    releaseDate: 2023-12-11
    lts: true
```

Reply:
yeah, all major versions are LTS releases until they are not supported, so I should add this flag to every release? (anyway, let me propose such change)